crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -149,7 +149,11 @@ def ceph_status
         return ret if ceph_nodes.empty?
         ceph_node = ceph_nodes.first
 
+        # FIXME: do I need to pick ceph-mon node? (It might happen that this is just ceph-osd, without the mon role)
         ssh_retval = ceph_node.run_ssh_cmd("LANG=C ceph health --connect-timeout 5 2>&1")
+        # FIXME: here we need some better check for health:
+        # a) could we ignore HEALTH_WARN in general, thus only checking for HEALTH_ERR?
+        # b) do we need some magic to suppress only some kind of warnings, making the cluster return HEALTH_OK?
```

Comment:
Nothing immediately useful from upstream (there's a three year old feature request for an [easier-to-process health report](http://tracker.ceph.com/issues/7192))